src/Stack/Build/Execute.hs

```diff
@@ -1949,7 +1961,7 @@ singleTest topts testsToRun ac ee task installedMap = do
                         "package-db " <> fromString snapDBPath <> "\n" <>
                         "package-db " <> fromString localDBPath <> "\n" <>
                         foldMap (\ghcId -> "package-id " <> RIO.display (unGhcPkgId ghcId) <> "\n")
-                            (thGhcId:M.elems allDepsMap)
```

Comment:
Why is `thGhcId` now excluded?

Reply:
what do you mean by "excluded"? I see it in `pkgGhcIdList ++ thGhcId:M.elems allDepsMap`

Reply:
Sorry, I'm stupid, ignore me. Totally misread the code.